template/package.json

```diff
@@ -1,20 +1,7 @@
 {
-  "name": "create-evergreen-app",
-  "version": "0.4.1",
-  "description": "Create an evergreen web application with no build configuration",
-  "license": "MIT",
-  "author": "Owen Buckley <owen@thegreenhouse.io>",
-  "main": "src/index.js",
-  "homepage": "https://github.com/ProjectEvergreen/create-evergreen-app#readme",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/ProjectEvergreen/create-evergreen-app.git"
-  },
+  "name": "my-app",
+  "version": "0.1.0",
   "private": true,
-  "keywords": [
-    "Web Components",
-    "Lit Element"
-  ],
```

Comment:
This is fine i guess if you want to do it this way instead. You should also remove the bin field.

Reply:
it's not there though.  ❓